qap.go

```diff
@@ -139,17 +144,48 @@ func randomWalk(assignment Assignment, timeLimit int64, m1, m2 IntMat) (Assignme
 	return bestAssignment, bestCost, stepCount, stop.Microseconds()
 }
 
+func measureTime(filename string, times int) {
+	m1, m2, _ := fileReader("instances/" + filename + ".dat")
+	var SArray, GArray, RWArray, RArray [][2]int
```

Comment:
czy one nie są długości `times`?
a jak chcesz jako slice to nie lepiej je make'ować?

Reply:
myślałem o tym, ale więcej pisania a gain zupełnie znikomy i irrelevant w osiąganym celu